Observer/Observer/ConsoleObserver/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Net.Mentoring.Patterns.ConsoleObserver
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ConsoleReader conReader = new ConsoleReader();
```

Comment:
![image](https://user-images.githubusercontent.com/22260621/48842499-1dc68d00-ed9d-11e8-9bad-9f620d7ddc88.png)
- Please fix spelling typos
- If you take into account input case - please tell the user this(or ignore case).